src/index.ts

```diff
@@ -1,6 +1,6 @@
 export enum TarFileType {
-  File,
-  Dir,
+  File = 0,
+  Dir = 5,
```

Comment:
The values should be the ascii code of `'0'` and `'5'`, which is `48` and `53`. Could you make an update?